src/components/Card/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react';
+
+export interface CardProps {
+  title?: string;
+  description?: string;
+  image?: string;
+  isLink?: boolean;
+  url?: string;
+  isExternal?: boolean;
+  children?: ReactNode;
+}
+
+const Card: React.FC<CardProps> = ({
+  title,
+  description,
+  image,
+  isLink,
+  url,
+  isExternal,
+  children,
+}: CardProps) => {
+  const content = (
+    <>
+      {image && <img src={image} className="w-full" />}
+      <div className="px-6 py-4">
+        {title && (
+          <h3 className="text-gray-800 text-3xl font-semibold">{title}</h3>
+        )}
+        {description && <div className="mt-2 text-gray-600">{description}</div>}
+      </div>
+      {children}
+    </>
+  );
+
+  return isLink ? (
+    <div
+      data-testid="card-element"
+      className="max-w-sm rounded overflow-hidden shadow-lg"
+    >
+      <a
+        href={url}
+        target={isExternal ? '_blank' : ''}
```

Comment:
Good point, @marianyfsilva

I'm considering these components to a separated library based on Windmill-React-UI. I initially created a fork [here](https://github.com/sumelms/windstorm) and called it Windstorm.

I still do some tests around it to make sure that it would be a good idea. Hopefully, I'll be able to finish it during the weekend. The library has already given us some basic components, and we can extend them to make them more useful for our needs.